### PR TITLE
Update LOG_WEB_CONTEXT_PARAMS options

### PR DESCRIPTION
**Pull request by volosied**
https://github.com/apache/myfaces/pull/520#issuecomment-1432835186

Updating to "always, dev-only, never" so that they describe the functionality better.  (Note: always only means dev or production and not the other stages)

### CODE REVIEW

**Comment by volosied** [comment]
We don't log anything for UnitTest and SystemTest. That should stay the same, right?

**Comment by bohmber** [comment]
I prefer keep true, false as well for backwards compatiblity. That means always, dev-only, never, true, false

**Comment by bohmber** [comment]
UnitTest, SystemTest good question. What about none-production or something similar

**Comment by volosied** [comment]
@bohmber I've updated the code again. Let me know what you think. 

The 5 options now are: always, true, dev-only, never, false, non-production

**Review by bohmber — APPROVED** [review]
For me it looks better now. Lets wait for @tandraschko comments

**Comment by volosied** [comment]
Spotted a new errors -- they're fixed now.

**Review by bohmber — APPROVED** [review]

File: impl/src/main/java/org/apache/myfaces/config/webparameters/MyfacesConfig.java [hunk 1]
```diff
@@ -714,14 +714,15 @@
     /**
      * Indicate if INFO logging of all the web config params should be done before initialize the webapp. 
      * <p>
-     * If is set in "auto" mode, web config params are only logged in "Development" mode. 
-     * If is set to "true", web config params are only logged in "Production" and "Development" mode. 
-     * If is set in "false" mode, no info logging occurs in either mode.
+     * If set to "dev-only" mode, web config params are only logged in "Development" mode. 
+     * If set to "true" or "always", web config params are only logged in "Production" and "Development" mode. 
+     * If set to "false" or "never", no info logging occurs in either mode.
```
Comment (volosied) [inline impl/src/main/java/org/apache/myfaces/config/webparameters/MyfacesConfig.java:719]: Maybe? 
```suggestion
     * If set to "false" or "never",  info logging does not occur in any mode.
```

Comment (bohmber) [reply impl/src/main/java/org/apache/myfaces/config/webparameters/MyfacesConfig.java:719]: That is up to you

**Comment by volosied** [comment]
No word from Thomas yet so I set it to dev-only here: https://github.com/apache/myfaces/commit/381f2714da9e32ea23a77df20db5983afcc7408d

Will start RC5 release now.

**Comment by tandraschko** [comment]
TBH it feels a bit overengineered now with to many options for such a small logging

But If its only logged in dev stage per default, im ok with it
Please dont forget the homepage

**Comment by volosied** [comment]
I agree with Thomas. I've already changed auto to dev-only, so I'll just close this PR.

**Pull request closed without merging**